custom_components/opnsense/update.py

```
@@ -31,111 +31,100 @@
+        except (TypeError, KeyError):
+            self._available = False
+            return
+        self._available = True
 
-    @property
-    def latest_version(self):
-        """Latest version available for install."""
-        state = self.coordinator.data
+        try:
+            self._attr_installed_version = dict_get(
+                state, "firmware_update_info.product.product_version"
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)